templates/terraform/examples/redis_instance_full.tf.erb

```diff
@@ -18,6 +18,6 @@ resource "google_redis_instance" "<%= ctx[:primary_resource_id] %>" {
   }
 }
 
-resource "google_compute_network" "auto-network" {
+data "google_compute_network" "auto-network" {
```

Comment:
auto-network as a name is probably outdated now since this isn't provisioning the network